scripts/release/02-execute-release.sh

```
@@ -0,0 +1,69 @@
+echo "=================================================="
+echo "EXECUTING AUTOMATED RELEASE FOR: $REPO_DIR"
+echo "=================================================="
+
+# Ensure workspace tree is clean and synchronized
+git fetch upstream main -q
+git checkout main -q
+git pull --ff-only upstream main -q
+
+# Step 1: Calculate NEXT_VERSION increment rule
+LATEST_TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo "v0")
+VERSION_NUM=$(echo "$LATEST_TAG" | sed 's/v//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)